schemas/com.snowplowanalytics.monitoring.batch/alert/jsonschema/1-0-0

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Webhook payload describing an alert coming from RDB Loader",
+  "self": {
+    "vendor": "com.snowplowanalytics.monitoring.batch",
+    "name": "alert",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "folder": {
+      "description": "A folder that caused an alert",
+      "type": "string",
```

Comment:
If `folder` is optional, I'd expect `["string", "null"]` here, wdyt?

Reply:
I noticed that `folder` isn't required hence optional, is there a strategy on picking this approach vs making it accept `"null"` ?

Reply:
Yeah, you're right. I expressed optionality by not mentioning it in `required` because circe encoders by default just skip the field, but I agree it's better to be explicit.